### PR TITLE
Frontend: RTL alignment fixes

### PR DESCRIPTION
**Pull request by haimkastner**
- Navigation sub-menu item margin 
- Notifier RTL text support
- Media list view headers direction
- Edit photo details view 
- Share content view 
- Upload media RTL text support

### CODE REVIEW

File: frontend/src/dialog/upload.vue [hunk 1]
```diff
@@ -10,7 +10,7 @@
           <translate key="Upload">Upload</translate>
         </v-toolbar-title>
       </v-toolbar>
-      <v-container grid-list-xs text-xs-left fluid>
+      <v-container grid-list-xs :text-xs-left="!rtl" :text-xs-right="!rtl" fluid>
```
Comment (StephenBrown2) [inline frontend/src/dialog/upload.vue:13]:
```suggestion
      <v-container grid-list-xs :text-xs-left="!rtl" :text-xs-right="rtl" fluid>
```

File: frontend/src/dialog/share.vue [hunk 2]
```diff
@@ -100,7 +100,7 @@
           </v-expansion-panel-content>
         </v-expansion-panel>
 
-        <v-container fluid text-xs-left class="pb-0 pt-3 pr-0 pl-0 caption">
+        <v-container fluid :text-xs-left="!rtl" :text-xs-right="!rtl" class="pb-0 pt-3 pr-0 pl-0 caption">
```
Comment (StephenBrown2) [inline frontend/src/dialog/share.vue:103]: Looks like this got copy-pasted...
```suggestion
        <v-container fluid :text-xs-left="!rtl" :text-xs-right="rtl" class="pb-0 pt-3 pr-0 pl-0 caption">
```

File: frontend/src/component/navigation.vue [hunk 3]
```diff
@@ -265,7 +265,7 @@
           </v-list-tile-action>
 
           <v-list-tile-content>
-            <v-list-tile-title>
+            <v-list-tile-title >
```
Comment (StephenBrown2) [inline frontend/src/component/navigation.vue:268]:
```suggestion
            <v-list-tile-title>
```

**Review by StephenBrown2 — COMMENTED** [review]
Looks like a couple `!rtl`s got copy-pasted...

**Comment by haimkastner** [comment]
Hi @StephenBrown2,

Yes, you're right about the "copy-paste". 

Most of the "vuetifyjs" components support RTL out-of-the-box.

But some of the styles are not supported RTL natively the and I didn't found a better way to implement it other than a high-risk CSS styles override.

Do you know a better way to that?

By the way, I already set the first RTL supported using this ugly way see #1220

**Comment by StephenBrown2** [comment]
> But some of the styles are not supported RTL natively the and I didn't found a better way to implement it other than a high-risk CSS styles override.

Ah, good to know. I'm not familiar with RTL or really Vuetify yet, so your guess is as good as mine.

**Comment by lastzero** [comment]
Is this good to merge?

**Comment by graciousgrey** [comment]
Any update on this?

**Comment by haimkastner** [comment]
Hi @graciousgrey,  
To the best of my knowledge, the current implementation is the best way to add RTL support without a risk to break something even that it's contained a few copy-pastes.

**Pull request merged**